Automattic/wp-calypso
# Multisite Scheduled Updates: Implement schedule activation toggle control in schedule list

Closes https://github.com/Automattic/wp-calypso/issues/90265

## Proposed Changes

* Implement batch activation toggle
* Update label IDs

## Testing Instructions



* Go to `/plugins/scheduled-updates`
* Make sure you have a scheduled updates on multiple sites
* Toggle batch activation and check if it works properly
* Toggle single site activation and check if it works properly

![Screen Capture on 2024-05-13 at 23-56-19](https://github.com/Automattic/wp-calypso/assets/1241413/22f58622-203f-4ee6-a6ac-9746b4ef28d9)


## Pre-merge Checklist
